uber/templates/registration/new.html

```diff
@@ -9,72 +11,76 @@
 
 <script type="text/javascript">
     // automatically log out after 15 minutes of inactivity
-    setTimeout(function() {
+    setTimeout(function () {
         window.location = "../accounts/logout";
     }, 900000);
     
     var UNASSIGNED = {{ unassigned|jsonize }};
     var groupChosen = function(group) {
         var $group = $(group);
-        var $link = $group.prev("a");
-        var $badgeNum = $group.nextAll(":text");
-        if ($group.val() == "") {
+        var $link = $group.prev('a');
+        var $badgeNum = $group.nextAll(':text');
+        if ($group.val() === '') {
             $link.hide();
```

Comment:
small note (and I'm so guilty of doing this entirely too often) but we should strive where possible to avoid doing real code changes inline with these sorts of stylistic cleanup (like single quotes to double quotes).  This is purely for code review purposes so that we're only focused in one review on either 1) the code changes or 2) the stylistic changes.  i.e. I might glance over something thinking it's a style cleanup but in reality it was actually a potentially functionality inducing change (like ```==``` to ```===``` here, which is still probably fine)